source/features/comment-fields-keyboard-shortcuts.tsx

```diff
@@ -86,5 +66,7 @@ features.add({
 }, {
 	waitForDomReady: false,
 	repeatOnAjax: false,
-	init
+	init: () => {
+		listenToCommentFields(init);
+	}
```

Comment:
Avoid doing this for any user action, as suggested here https://github.com/sindresorhus/refined-github/pull/3119#discussion_r428759134

Key/click listeners should not be named `init`. Revert here and other related places (including `keyup`)